packages/preview-middleware/src/base/flp.ts

```diff
@@ -147,6 +141,14 @@ export class FlpSandbox {
      * Add routes for html and scripts required for a local FLP.
      */
     private addStandardRoutes() {
+        // add route for the oninit script
+        this.router.get(`${dirname(this.config.path)}/sandboxInit.js`, (_req: Request, res: Response) => {
```

Comment:
Please change to `${dirname(this.config.path)}sandboxInit.js`. This will fix the issue that I reported.

Reply:
No, because for a config path like `/my/path/index.html` it won't work but https://github.com/SAP/open-ux-tools/pull/1256/commits/cc5881a85b85e261d7bbff0cb4b6a0cfa4207efe should fix it